queue/group_remote.go

```diff
@@ -286,7 +300,7 @@ func (g *remoteQueueGroup) Prune(ctx context.Context) error {
 					catcher.Add(err)
 					return
 				}
-				if err := one.Decode(struct{}{}); err != nil {
+				if err := one.Decode(&struct{}{}); err != nil {
```

Comment:
given that we don't care about the result, shouldn't we just be doing counts here?

Reply:
done